stable/datadog/templates/container-trace-agent.yaml

```diff
@@ -7,7 +7,7 @@
 {{ toYaml .Values.agents.containers.traceAgent.resources | indent 4 }}
   ports:
   - containerPort: 8126
-    {{- if .Values.agents.useHostPort }}
+    {{- if .Values.datadog.apm.enabled }}
```

Comment:
I think you can remove this condition since the "container-trace-agent" is added in the daemonset.yaml file only if `.Values.datadog.apm.enabled` is true
https://github.com/clamoriniere/charts/blob/feature/datadogv2/stable/datadog/templates/daemonset.yaml#L63